### PR TITLE
Migrate resources.php (+ update)

### PR DESCRIPTION
**Pull request by mlep**
In response to Issue https://github.com/JabRef/www.jabref.org/issues/34.

Please review the content. I included exclamation points (!!!) when in doubt.

Also, I am not sure where this file should be on the website...

### CODE REVIEW

**Comment by Siedlerchr** [comment]
It's okay to have it as new section.
Some hints:
bg: The background color, e.g. blue for features, green for donations
style: center
fa-icon: Name of the One font-icon from here: http://fontawesome.io/

The other color is probably the text color

**Comment by Siedlerchr** [comment]
The "color" attribute is specificed in the _config.yml

**Comment by mlep** [comment]
@Siedlerchr Thank you for pointing out where I could find the parameters. I have added them.

File: _sections/60-Ressources.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+*   OK
+
+If you now go to **File -&gt; Export**, Jabref will open a window where you can choose different types of exportfilters below. Select **Files of type: IJoGGC(*.rtf)**.
+
+### APA export filter
+
+[Russell Almond's APA export filter](http://ralmond.net/APAish/) provides an export filter that is close to the American Pychological Association (APA) guidelines. It exports the references in Rich Text Format (rtf).
+
+### CSV export filter with Mendeley tags
+[Cynthia D'Angelo's CSV export filter with Mendeley tags](https://cynthiadangelo.com/2012/10/22/exporting-mendeley-tags/) provide an export filter that allows for exporting Mendely tags. It exports the refencence in CVS format (i.e. spreadsheet- compatible).
```
Comment (Siedlerchr) [inline _sections/60-Ressources.md]: last sentence:  CVS -> csv format `?

File: _sections/60-Ressources.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+### Feinerleiser
+
+**!!!Status?**
+
+_by Niklas Alt_
+
+Feinerleiser is a tool for improving the JabRef-LibreOffice integration when writing for the Humanities. This tool can be run to finalize a document, providing citation features that are not supported by JabRef itself.
+
+[Feinerleiser](http://www.sourceforge.net/projects/feinerleiser/)
```
Comment (Siedlerchr) [inline _sections/60-Ressources.md]: As the last update is from 2013, I doubt that it is still actual. But when I see it right it as external tool, which does not depend on Jabref itself but only on OO and

Comment (mlep) [inline _sections/60-Ressources.md]: OK, let's remove it, we will see if somebody asks for its inclusion

**Comment by mlep** [comment]
No more exclamation points. Thanks @Siedlerchr  for your review.

**Comment by mlep** [comment]
Centered text does not lead to a convenient reading...

**Comment by Siedlerchr** [comment]
Yeah, maybe we can set it too left

**Comment by Siedlerchr** [comment]
I changed it to left, but does not rebuild I think.
Will check it later. Have no ruby installed here

2016-10-13 16:00 GMT+02:00 MLEP <notifications@github.com>:

**Comment by Siedlerchr** [comment]
@mlep  Fixed the icon. You just had to omit the fa-prefix.
But we have to change the bg color to sth other.

**Comment by mlep** [comment]
@Siedlerchr Oops, sorry for the prefix...
Indeed, for the Resources, section, the icon and title are in the background color. The strange thing is that for the Help section, the background is in a clear color, but the icon and title are dark...

**Pull request merged**